### PR TITLE
Fix veth pair

### PR DESCRIPTION
**Pull request by edwarnicke**

### CODE REVIEW

File: pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go [hunk 1]
```diff
@@ -43,13 +43,13 @@ func appendInterfaceConfig(ctx context.Context, conn *networkservice.Connection,
 		}
 		conf.GetLinuxConfig().Interfaces = append(conf.GetLinuxConfig().Interfaces,
 			&linuxinterfaces.Interface{
-				Name:       name,
+				Name:       name + "-veth",
 				Type:       linuxinterfaces.Interface_VETH,
 				Enabled:    true,
 				HostIfName: linuxName,
 				Link: &linuxinterfaces.Interface_Veth{
 					Veth: &linuxinterfaces.VethLink{
```
Comment (denis-tingajkin) [inline pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go:51]: Also, I would recommend adding these lines from https://github.com/networkservicemesh/networkservicemesh/blob/master/forwarder/vppagent/pkg/converter/kernel_connection_converter.go#L126
```suggestion
				Veth: & linuxinterfaces.VethLink{
					PeerIfName:          name,
					RxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
					TxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
				},
```
this fixes some bugs with packet sending with latest vpp agent

Comment (edwarnicke) [reply pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go:51]: Done :)

File: pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go [hunk 2]
```diff
@@ -64,7 +64,9 @@ func appendInterfaceConfig(ctx context.Context, conn *networkservice.Connection,
 				},
 				Link: &linuxinterfaces.Interface_Veth{
 					Veth: &linuxinterfaces.VethLink{
-						PeerIfName: linuxName,
+						PeerIfName:           name + "-veth",
+						RxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
+						TxChecksumOffloading: linuxinterfaces.VethLink_CHKSM_OFFLOAD_DISABLED,
```
Comment (denis-tingajkin) [inline pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go:69]: Do we need to apply these changes for line 51?

Comment (edwarnicke) [reply pkg/networkservice/mechanisms/kernel/kernelvethpair/common.go:71]: Good catch, done :)

**Pull request merged**